src/Bundle/DependencyInjection/DunglasDoctrineJsonOdmExtension.php

```diff
@@ -50,5 +52,15 @@ public function load(array $configs, ContainerBuilder $container): void
         if (!class_exists(BackedEnumNormalizer::class) || !class_exists(\BackedEnum::class)) {
             $container->removeDefinition('dunglas_doctrine_json_odm.normalizer.backed_enum');
         }
+
+        $config = $this->processConfiguration(new Configuration(), $configs);
+
+        $typeMapConfig = $config['types'];
+        if ($typeMapConfig) {
+            $container->setDefinition(
```

Comment:
Usually we prefer defining the service in the XML file and removing the definition with `removeDefinition()` in the extension. This allow to have service auto-completion in PHPStorm and the like.